fuse/community/aggregation/booster.yaml

```diff
@@ -0,0 +1,15 @@
+name: Red Hat Fuse - Aggregation Example
+description: Booster to expose an HTTP REST endpoint using Apache Camel where the message aggregated from 2 services.
+source:
+  git:
+    url: https://github.com/integr8ly/fuse-rest-http-booster
```

Comment:
@david-martin Is this now meant to point at https://github.com/integr8ly/fuse-flights-aggregator ?